src/evict/evict_lru.c

```diff
@@ -2156,11 +2175,28 @@ __evict_walk_tree(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue, u_int max_ent
         if (F_ISSET(ref, WT_REF_FLAG_INTERNAL))
             internal_pages_queued++;
 
-        __wt_verbose(session, WT_VERB_EVICTSERVER, "walk select: %p, size %" WT_SIZET_FMT,
-          (void *)page, page->memory_footprint);
+        if (WT_VERBOSE_LEVEL_ISSET(session, WT_VERB_EVICTSERVER, WT_VERBOSE_DEBUG_2)) {
+            if (urgent_queued_flag)
```

Comment:
I was thinking that we only need to aggregate the number of pages and sizes. We don't need to print out the address of pages. Do you find the address useful for debugging?